officers.md

```diff
@@ -9,26 +9,26 @@ permalink: /officers/
 ## <a name="Officers"></a>Officers<a href="#Officers"><i class="fa fa-link" aria-hidden="true"></i></a>
 
 
-### <a name="Danny">[Danny Manesh](https://maneshd.github.io/), President<a href="#Danny"><i class="fa fa-link" aria-hidden="true"></i></a>
+### <a name="Wesley">[Wesley Woo](https://wesmwoo.github.io), President<a href="#Wesley"><i class="fa fa-link" aria-hidden="true"></i></a>
 
 Danny has taken control from Alex.
```

Comment:
Wesley has taken control from Danny? >:)

Reply:
👀

Reply:
Also feel free to change the little blurbs (or offer that the new officers change their own)